### PR TITLE
added script to create indices 

### PR DESCRIPTION
**Pull request by bogensberger**

### CODE REVIEW

File: blackbox/bwc/create_bwc_index.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+CRATE_HTTP_PORT = '42222'
+CRATE_TRANSPORT_PORT = '43333'
+
+
+CREATE_TABLE = """
+    CREATE TABLE legacy_geo_point (
+         id int primary key,
+         p geo_point) clustered into 1 shards
```
Comment (chaudum) [inline blackbox/bwc/create_bwc_index.py]: strange formatting. better:

```sql
CREATE TABLE legacy_geo_point (
     id INT PRIMARY KEY,
     p GEO_POINT
) CLUSTERED INTO 1 SHARDS WITH (
    number_of_replicas=0, 
    "translog.flush_threshold_ops"=0, 
    "gateway.local.sync"=0
)
```

File: blackbox/bwc/create_bwc_index.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+         WITH (number_of_replicas=0, 
+               "translog.flush_threshold_ops"=0, 
+               "gateway.local.sync"=0
+        )
+    """
+
+INSERT = """
+    INSERT INTO legacy_geo_point (id, p) VALUES (1, 'POINT (10 10)')
```
Comment (chaudum) [inline blackbox/bwc/create_bwc_index.py]: do we need only a single record?

Comment (bogensberger) [inline blackbox/bwc/create_bwc_index.py]: yes, for the current test we only need a single record

File: blackbox/bwc/create_bwc_index.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+def create_index(cfg):
+    crate_layer = CrateLayer(
+        'data',
+        crate_home=cfg.crate_home,
+        port=CRATE_HTTP_PORT,
+        transport_port=CRATE_TRANSPORT_PORT
+    )
+    crate_layer.start()
+    try:
+        conn = connect('localhost:' + CRATE_HTTP_PORT)
```
Comment (chaudum) [inline blackbox/bwc/create_bwc_index.py]:
```py
with conn = connect('...'):
    cur = conn.cursor()
    ...
```

Comment (chaudum) [inline blackbox/bwc/create_bwc_index.py]: this will close the connection automatically when done

**Comment by chaudum** [comment]
would it be possible to make the bwc index creation a bit more generic?

File: blackbox/bootstrap.sh [hunk 4]
```diff
@@ -3,7 +3,7 @@
 # travis and jenkins have python3.4 installed
 if hash python3.4 2> /dev/null; then
     python3.4 -m venv .venv --without-pip
-    wget https://bootstrap.pypa.io/get-pip.py
+    wget https://bootstrap.pypa.io/get-pip.py --no-check-certificate
```
Comment (mfussenegger) [inline blackbox/bootstrap.sh]: shouldn't pypa.io have a valid certificate? Why is this required?

**Comment by chaudum** [comment]
ack if travis is green
